File: 2022/day1/Python/Main.py
data = open(".\\..\\input.txt", "r").read().splitlines()
totalCalories, highestCalories = 0, 0
for elfCalories in data:
    totalCalories = 0 if (elfCalories == "") else (totalCalories + int(elfCalories))
    if(totalCalories > highestCalories): highestCalories = totalCalories
print(highestCalories)

File: 2022/day1/C++/Main.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;
int main()
{
    ifstream file(".\\..\\input.txt");
    if (!file.is_open())
        throw runtime_error("could not open file");
    string fileLine;
    int highestCals, calories;
    while (getline(file, fileLine)){
        if(calories > highestCals) highestCals = calories;
        calories = (fileLine.empty()) ? 0 : calories + stoi(fileLine);
    }
    file.close();
    cout << highestCals << endl;
    return 0;
}